app/workers/articles/detect_human_language_worker.rb

```diff
@@ -0,0 +1,14 @@
+module Articles
+  class DetectHumanLanguageWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform(article_id)
+      article = Article.find_by(id: article_id)
+      return unless article
+
+      article.update_column(:language, LanguageDetector.new(article).detect)
```

Comment:
This being an `update_column` and the fact that [this language detector gem](https://github.com/jtoy/cld) seems like it is pretty fast. I am wondering if we even need this in a job. I think we could do it inline. @rhymes or @maestromac any thoughts?

Reply:
I just generated a random 99 paragraph English text (~5.7k words/32k characters) and the gem's response was basically instant. So while it goes a bit against my natural tendency of moving things off the main thread, I guess we could indeed inline this. If we want to be extra careful we could use Rail's `String#truncate` to limit ourselves to a certain amount of characters, after all the gem's documentation uses pretty short sentences too.